mmv1/products/beyondcorp/api.yaml

```diff
@@ -0,0 +1,252 @@
+      remote enterprise environment and GCP. It initiates connections to the applications and can proxy the 
+      data from users over the connection.
+    min_version: ga
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Official Documentation': 'https://cloud.google.com/beyondcorp-enterprise/docs/enable-app-connector'
+      api: 'https://cloud.google.com/beyondcorp/docs/reference/rest#rest-resource:-v1.projects.locations.appconnectors'
+    base_url: projects/{{project}}/locations/{{region}}/appConnectors
+    self_link: projects/{{project}}/locations/{{region}}/appConnectors/{{name}}
+    create_url: projects/{{project}}/locations/{{region}}/appConnectors?app_connector_id={{name}}
+    delete_url: projects/{{project}}/locations/{{region}}/appConnectors/{{name}}
+    update_url: projects/{{project}}/locations/{{region}}/appConnectors/{{name}}
+    update_verb: :PATCH
+    update_mask: true
+    # iam_policy: !ruby/object:Api::Resource::IamPolicy
+    #   parent_resource_attribute: name
+    #   method_name_separator: ':'
+    #   fetch_iam_policy_verb: :GET
+    async: !ruby/object:Api::OpAsync
+      operation: !ruby/object:Api::OpAsync::Operation
+        path: 'name'
+        base_url: '{{op_id}}'
+        wait_ms: 1000
+      result: !ruby/object:Api::OpAsync::Result
+        path: 'response'
+        resource_inside_response: true
+      status: !ruby/object:Api::OpAsync::Status
+        path: 'done'
+        complete: true
+        allowed:
+          - true
+          - false
+      error: !ruby/object:Api::OpAsync::Error
+        path: 'error'
+        message: 'message'
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'name'
+        required: true
+        input: true
+        description: |
+          ID of the AppConnector.
+      - !ruby/object:Api::Type::String
+        name: 'region'
+        description: |
+          The region of the AppConnector.
+        input: true
+        url_param_only: true
+      - !ruby/object:Api::Type::String
+        name: 'displayName'
+        description: |
+          An arbitrary user-provided name for the AppConnector.
+      - !ruby/object:Api::Type::KeyValuePairs
+        name: 'labels'
+        description: |
+          Resource labels to represent user provided metadata.
+      - !ruby/object:Api::Type::NestedObject
+        name: 'principalInfo'
+        description: |
+          Principal information about the Identity of the AppConnector.
+        required: true
+        properties:
+          - !ruby/object:Api::Type::NestedObject
+            name: 'serviceAccount'
+            description: |
+              ServiceAccount represents a GCP service account.
+            properties:
+              - !ruby/object:Api::Type::String
+                name: 'email'
+                description: |
+                  Email address of the service account.
+      - !ruby/object:Api::Type::Time 
+        name: 'createTime'
```

Comment:
Are these output only fields useful? We try to omit them when we don't expect any user to use them, like create & update time